config/clusters/awi-ciroh/common.values.yaml

```diff
@@ -51,16 +51,13 @@ basehub:
             - NOAA-OWP
           scope:
             - read:org
-    # FIXME: The configurator is used, so we should not be using prePuller at
-    #        the same time. This is just used to help pre-warm nodes during an
-    #        event.
-    #
-    prePuller:
-      continuous:
-        enabled: true
     singleuser:
```

Comment:
Maybe we should minimize the changes in this PR to the node updates -  is there any harm in just match the configurator image `quay.io/2i2c/awi-ciroh-image:584293e50d4c` and continuing to prepull for the event to  decrease startup time.

Reply:
Fine ignore this comment. However this sounds like our system the owner of the image (configurator/kubespawner) should control the prepulling not the helm chart longer term.